### ISSUE TITLE
Infinite loop on INFO: Retrieving F1656F24C74CD1D8 key..

### ISSUE THREAD

**Issue by lexsoft00** [omega8cc/boa#1323]
Hi,

I've tried instaling the latest BOA head on a new server ( Debian 9 ) with PHP 7.1

 It's stuck with an infinite loop:

```
 default: BOA [02:40:32] ==> INFO: Installing MariaDB...
    default: BOA [02:40:47] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [02:40:50] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [02:40:52] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [02:40:54] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [02:40:56] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [02:40:58] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [02:41:01] ==> INFO: Retrieving F1656F24C74CD1D8 key..
```

```
    default: BOA [03:00:38] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [03:00:40] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [03:00:42] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [03:00:44] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [03:00:46] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [03:00:49] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [03:00:51] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [03:00:53] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [03:00:55] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [03:00:57] ==> INFO: Retrieving F1656F24C74CD1D8 key..
    default: BOA [03:00:59] ==> INFO: Retrieving F1656F24C74CD1D8 key..
```

**Comment by omega8cc** [comment]
When you have tried it for the last time? We couldn't reproduce this now..

**Comment by lexsoft00** [comment]
I've tried it today. I will rebuild the server and try again later today.

**Comment by omega8cc** [comment]
It should be fixed already.

**Comment by tourtools** [comment]
same problem right now:

```
BOA [02:52:02] ==> INFO: Installing MariaDB...
BOA [02:52:07] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:09] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:11] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:13] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:15] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:17] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:19] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:21] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:23] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:26] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:28] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:30] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:32] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:34] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:36] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:38] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:40] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:42] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:44] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:46] ==> INFO: Retrieving F1656F24C74CD1D8 key..
BOA [02:52:48] ==> INFO: Retrieving F1656F24C74CD1D8 key..
```

already rebuilt the server and same behaviour

**Comment by tourtools** [comment]
it was a pgp key refresh  issue

with a `killall dirmngr` all went at its place

**Comment by omega8cc** [comment]
We have just reproduced this too (with head).

**Comment by cwaone** [comment]
Hi, tried the last 2 days to install boa head on fresh debian 9 stretch - no success - almost landing in the failure mentioned above - any tipps?

**Comment by omega8cc** [comment]
Fixed in 64a2edd29822afeab01e821d637fa5118ecb3764 last year.
